testdata/goodConfig_fixture.yml

```diff
@@ -0,0 +1,21 @@
+---
+log_level: trace
+cache:
+  schema_file: testingtesting.schema.json
+endpoint: https://api-staging.newrelic.com/graphql
```

Comment:
Does this have to be staging?

Reply:
No, I just want it to be non-default, so we know its being loaded from this config.

Reply:
Updated.